src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -66,27 +66,32 @@ object Messages {
   }
 
   private def sendSlackMessage[T <: IncomingEvent, TContext](incomingEvent: T, context: TContext)(implicit messages: Messages[T, TContext]): Option[IO[String]] = {
-    messages.slack(incomingEvent, context).map(slackMessage => {
-      val url = incomingEvent match {
-        case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "judgment") =>
-          eventConfig("slack.webhook.judgment_url")
-        case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "standard") =>
-          eventConfig("slack.webhook.standard_url")
-        case _: ExportStatusEvent => eventConfig("slack.webhook.export_url")
-        case _: KeycloakEvent => eventConfig("slack.webhook.tdr_url")
-        case _ => eventConfig("slack.webhook.url")
-      }
-      val requestBody = slackMessage.asJson.noSpaces
+    val urls = webhookUrlsForEvent(incomingEvent)
+    messages.slack(incomingEvent, context).map { message =>
       AsyncHttpClientCatsBackend.resource[IO]().use { backend =>
-        val request = basicRequest
-          .post(uri"$url")
-          .body(requestBody)
-          .contentType(MediaType.ApplicationJson)
-        for {
-          response <- backend.send(request)
-          body <- IO.fromEither(response.body.left.map(e => new RuntimeException(e)))
-        } yield body
+        urls.traverse { url =>
+          backend.send(buildRequest(message, url)).flatMap { response =>
+            IO.fromEither(response.body.left.map(e => new RuntimeException(e)))
+          }
+        }.map(_.mkString("\n"))
       }
-    })
+    }
+  }
+  
+  private def buildRequest(message: SlackMessage, webhookUrl: String) =
+    basicRequest.post(uri"$webhookUrl").body(message.asJson.noSpaces).contentType(MediaType.ApplicationJson)
+      
+  private def webhookUrlsForEvent[TContext, T <: IncomingEvent](incomingEvent: T): Seq[String] = {
+    incomingEvent match {
+      case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "judgment") =>
+        Seq(eventConfig("slack.webhook.judgment_url"))
+      case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "standard") =>
+        Seq(eventConfig("slack.webhook.standard_url"))
+      case ev: ExportStatusEvent =>
+        val failureEscalationUrl = Option.when(ev.environment == "prod" && !ev.success)(eventConfig("slack.webhook.tdr_url"))
+        Seq(Some(eventConfig("slack.webhook.export_url")), failureEscalationUrl).flatten
```

Comment:
Wondering if it would make more sense to send the failure notification to the new channel standard export channel Vimlesh created `slack.webhook.export_url` instead of `slack.webhook.export_url` so it's all in once place?

Reply:
I suppose the only thing with that is there isn't an obvious way of separating standard failures from judgment failures (unless I've missed it :smile:) as the way we're set up at the moment has the `consignmentType` stored in the success details. I think it might be worth a card anyway to look into what it would take to refactor things so we get consignment types with failures too.

Reply:
Ahh yes I forgot the judgment fails still go to the usual channel 🤦‍♂️